osbs/cli/main.py

```diff
@@ -306,7 +306,7 @@ def cmd_build(args, osbs):
     elif args.orchestrator:
         create_func = osbs.create_orchestrator_build
     else:
-        create_func = osbs.create_prod_build
```

Comment:
I'm not sure about this line, @others?

Reply:
I don't think there should be 'else'

there should be just worker & orchestrator build, that 'create_build' is used for koji-containerbuild when orchestrator isn't enabled or older osbs-client doesn't have create_orchestrator_build method

Reply:
Should we default to some of the options though (e.g, orchestrator build)? If not, we should raise an error if neither args are passed here.

Reply:
prod argument should be removed as well
https://github.com/projectatomic/osbs-client/blob/6e337b285f50d44e8526a779e202cd9663400011/osbs/cli/main.py#L723

mmm maybe we can default to orchestrator instead of raising

https://github.com/projectatomic/osbs-client/blob/6e337b285f50d44e8526a779e202cd9663400011/osbs/cli/main.py#L721
default=True